src/main/java/org/synyx/urlaubsverwaltung/statistics/vacationoverview/api/VacationOverviewController.java

```diff
@@ -18,23 +22,33 @@
 
     private final VacationOverviewService vacationOverviewService;
 
+    private final AbsenceController absenceController;
+
     @Autowired
-    VacationOverviewController(VacationOverviewService vacationOverviewService) {
+    VacationOverviewController(VacationOverviewService vacationOverviewService, PersonService personService, ApplicationService applicationService, SickNoteService sickNoteService) {
         this.vacationOverviewService = vacationOverviewService;
+        this.absenceController = new AbsenceController(personService, applicationService,  sickNoteService);
     }
 
     @ApiOperation(
             value = "Get Vacation-Overview Metadata",
             notes = "Get Vacation-Overview metadata for all members of a department")
     @GetMapping("/vacationoverview")
-    public ResponseWrapper<VacationOverviewResponse> getHolydayOverview(
+    public ResponseWrapper<VacationOverviewResponse> getHolidayOverview(
             @RequestParam("selectedDepartment") String selectedDepartment,
             @RequestParam("selectedYear") Integer selectedYear,
             @RequestParam("selectedMonth") Integer selectedMonth) {
 
         List<VacationOverview> holidayOverviewList =
                 vacationOverviewService.getVacationOverviews(selectedDepartment, selectedYear, selectedMonth);
 
+        // to provide an efficient facade, also pull in the absences that would otherwise require separate API calls
+        for (VacationOverview v: holidayOverviewList){
+            v.setAbsences(
+                absenceController.personsVacations(selectedYear, selectedMonth, v.getPersonID(), null).getResponse().getAbsences()
```

Comment:
This is an absolute no-go! You managed to cross the architecture vertically AND horizontally. 👏